src/aho-corasick.cpp

```diff
@@ -427,6 +428,9 @@ Traverse the trie of *ac* from the root using suffix links where necessary.
 
 This function performs the same as ``traverse_word(ac, AhoCorasick.root, w)``
 )pbdoc");
+
+    // TODO(doc)
+    m.def("dot", [](AhoCorasick& ac) { return aho_corasick::dot(ac); });
```

Comment:
Do we want to call this `aho_corasick_dot` to simulate the namespace, and then `from ... import aho_corasick_dot as dot`? At the moment, the `AhoCorasick` doc contains the Dot doc for both `WordGraph` as well:
![image](https://github.com/user-attachments/assets/8de1b047-1c3e-4e57-ace9-181369c29e8e)

Reply:
Done